vendor/vendor.json

```diff
@@ -3,7 +3,7 @@
 	"ignore": "test",
 	"package": [
 		{
-			"checksumSHA1": "G5b4mSiqhTHSZW7W8aNKTrqe1xU=",
+			"checksumSHA1": "B9ZamUYiZjDG5gWtYLLxZ/oIaZo=",
```

Comment:
should we be using other kind of checksumSHAXXX in the near future? 
not a security issue but maybe a good practice?

Reply:
Possibly, but I'm not sure if that is supported by govendor.